src/web/js/trove/world-lib.js

```diff
@@ -750,10 +759,20 @@
                 k2();
               } else {
                 if (extras.closeWhenStop) {
-                  if (extras.closeBigBangWindow) {
-                    extras.closeBigBangWindow();
+                  if (lif) {
+                    runtime.ffi.throwMessageException('Cannot have both last-image and close-when-stop(true)');
```

Comment:
Sorry. Am involved in updating Patch to the latest horizon. I'll get back to testing your program when I can leave the update at a stable point.